tests/functional/test_home.py

```diff
@@ -41,15 +45,16 @@ def test_callouts(base_url, selenium):
 @pytest.mark.smoke
 @pytest.mark.nodata
 @pytest.mark.nondestructive
+@pytest.mark.maintenance_mode
 def test_header_displays(base_url, selenium):
     page = HomePage(selenium, base_url).open()
-    assert page.Header.is_displayed
-    assert page.Header.is_menu_displayed
```

Comment:
My original intention was simply to fix the bug, i.e. change "page.Header.is_menu_displayed" to "page.header.is_menu_displayed", but after making that change I discovered that the "is_menu_displayed" property of the header simply doesn't work (it searches for an element with an id of "nav-main-menu", but there is no such element on any of our pages), so I removed all calls to "page.header.is_menu_displayed" in all of our tests and removed the property.